src/query.ts

```diff
@@ -0,0 +1,36 @@
+import useSWR from "swr";
+import { configureBaseQueryHook } from "./query-base.js";
+
+/**
+ * ```ts
```

Comment:
Maybe worth adding a description to the function commands and a `@see` which links to the SWR docs for this